documentation/Get-Bootchart-metrics.md

```diff
@@ -0,0 +1,84 @@
+# Get Bootchart from a Clear Container #
+
+The following steps describe how to get a bootchart from a Clear Containers Container.
+Currently `systemd` is the default init process in the Clear Containers mini-os
+system. The Systemd project provides a tool called `systemd-bootchart` that
+collects the CPU load, disk load, memory usage, as well as per-process information
+from the os boot process.
+
+## Configure Clear Containers to get a bootchart ##
+
+To get a bootchart from a Clear Container, follow the steps below.
+
+### Prepare your Clear Container image ###
```

Comment:
This should not be a heading. Use ordered lists for steps. The best practice dictates new command requires its own step. Minor commands can be grouped together into a single step. Things like make a new directory and move into it, for example.